### PR TITLE
SuggestCommentScore using MD Bootstrap

### PR DESCRIPTION
**Pull request by chrismohrii**
Lets the user suggest feedback to Perspective using the SuggestCommentScore endpoint

Screenshot 2020-07-27 at 3 34 11 PM
Screenshot 2020-07-27 at 3 34 40 PM
Screenshot 2020-07-27 at 3 35 00 PM

### CODE REVIEW

**Review by danieldortega77 — APPROVED** [review]
LGTM - one note is that the post() function in SuggestCommentServlet.java could be removed once merged to master by using the ApiCaller.post() method in a PerspectiveCaller object

**Comment by chrismohrii** [comment]
> LGTM - one note is that the post() function in SuggestCommentServlet.java could be removed once merged to master by using the ApiCaller.post() method in a PerspectiveCaller object

Got it!

**Comment by olu144** [comment]
LGTM - nice color choice!

**Review by olu144 — APPROVED** [review]

File: capstone/src/main/webapp/script.js
Comment (dborkan) [inline capstone/src/main/webapp/script.js]: Is it still possible to use `TokenizerEnum['WORD']` here?

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java
Comment (dborkan) [inline capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: I think this is redefined in the `post` method, so maybe you can keep only 1 of these?  Also I'd suggest naming it something like `JSON_MEDIA_TYPE` just because there are multiple other json related variables, e.g. `String json`

File: capstone/src/main/webapp/script.js
Comment (dborkan) [inline capstone/src/main/webapp/script.js]: Do you want to change this to type `number`?  Then I think you can also specify a min and max value (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number)

**Review by dborkan — COMMENTED** [review]
Code mostly looks good, just a few small comments

**Review by dslucas — APPROVED** [review]

File: capstone/src/main/webapp/script.js
Comment (chrismohrii) [reply capstone/src/main/webapp/script.js]: These are actually not supposed to work 100% the same because of the way I fully preserve the format of the input (newlines, spaces, characters between "segments"). I haven't been able to find a solution where they are identical. I updated the code with the correct regular expressions, but I do realize that before I could have made your change before because `TokenizerEnum['WORD']` and `/(\S+\s*)/g` were identical.

File: capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java
Comment (chrismohrii) [reply capstone/src/main/java/com/google/sps/servlets/SuggestCommentServlet.java]: Yup, I removed this one. I'll hold off on changing the variable name in `post` because I entirely remove the `post` method from this file in the branch where I write the tests, but I've changed the variable name there to `jsonMediaType`

File: capstone/src/main/webapp/script.js
Comment (chrismohrii) [reply capstone/src/main/webapp/script.js]: Thanks, didn't know about this! I switched it (kept the error checking because I can't stop the user from entering some weird things)

**Review by dborkan — APPROVED** [review]
LGTM

**Pull request merged**
